docs/scripting.md

```diff
@@ -3,7 +3,7 @@ permalink: /docs/scripting/index.html
 layout: docs
 ---
 
-Hubot out of the box doesn't do too much, but it is an extensible, scriptable robot friend. There are [hundreds of scripts written and maintained by the community](/docs/#scripts.md) and it's easy to write your own.
+Hubot out of the box doesn't do too much, but it is an extensible, scriptable robot friend. There are [hundreds of scripts written and maintained by the community](/docs/#scripts.md) and it's easy to write your own.  You can either put together a quick script under your hubot's `scripts` directory or [create a script package](#creating-a-script-package) of your own!
```

Comment:
I don't know if I'd call the scripts living under `scripts` 'quick' :grin: They are some of our most complex ones, that aren't easily shareable.

Maybe phrase this like "You can create a custom script in hubot's `scripts` directory, or create a script package for sharing with the community"